### ISSUE TITLE
Cottle Script {Pause(xxxx)} make TTS distorded (solution found)

### ISSUE THREAD

**Issue by Ekristoffe** [EDCD/EDDI#2243]
## What's Wrong (please be as specific as possible)
There is a problem when in the default cottle script a pause is used, depending on the length of the pause the next TTS is distorted.
The longer the pause the grater the distortion happen. (especially in the FSD engaged script).

### Expected
I expect clear TTS even after a 5second pause.

### Observed
For example the "Destination confirmed","Jump in progress","Telemetry obtained" in the FSD engaged script will be distorted after the Pause(5000).

### Steps to reproduce
Just jump.
This may be exaggerated depending on the voice used (some accent seems to be more pause sensitive).

### Configuration
* Version [Please give full version numbers here, not just "latest" or "current"]
  * EDDI Version: 4.0.1
* Localization [if applicable]
  * EDDI's language: English
  * Voice: Microsoft Zira at first now using Microsoft Eva Mobile (Both english)

### Investigation Notes
Replacing all the:
{Pause(xxxx)}
by 
{Pause(xxxx)}.
resolve the problem.
Since the "," seems to reset the TTS state.
(The solution have been tested with all microsoft voice for :
English (US, UK, CA, AUS)
French (FR, CA)
Japanese (JP) (this one was just for fun with the JapEnglish pronunciation) ...

**Comment by Darkcyde13** [comment]
Hmm, I am unable to replicate this problem.

I've tried the default FSD engaged script (5 seconds) with the following voices (all British, apart from Zira which is American):
Ivona Amy
Microsoft George
Microsoft Hazel
Microsoft Susan
Microsoft Zira Desktop

I then tried them all with 10 seconds, and then 15 seconds.  None of them caused any distortion problems, and sounded exactly as I would expect, and exactly as the 'test voice' button in the Text-to-speech tab.

I'm sorry, I know this doesn't help your situation at all, but at least it shows that it's not affecting everyone, and I really wish I was able to help you further.

**Comment by Tkael** [comment]
This is not a problem that I'm familiar with either. Do you think that you could capture some audio samples to demonstrate the problem?

**Comment by Tkael** [comment]
(Providing as much detail as you can about the settings and scripts you use to create the samples so that we can try to replicate the issue?)

**Comment by Ekristoffe** [comment]
I have done more testing and strangely it happens only with eva now ... 
I have done a windows update yesterday ...
Here are the files,
This one is the original script:
[EDDI Original.zip](https://github.com/EDCD/EDDI/files/7029418/EDDI.Original.zip)
This one is the script with the comma added after the pause:
[EDDI Edited.zip](https://github.com/EDCD/EDDI/files/7029420/EDDI.Edited.zip)
This are the TTS settings
![image](https://user-images.githubusercontent.com/5562219/130396924-bd62f90d-13b5-44cf-8505-651d2dba4ca3.png)

Maybe the problem came from the voice itself (It's Cortana original voice) ...

**Comment by Tkael** [comment]
Thanks, that's excellent data. I believe that this is a duplicate of an old issue, #445.

Each voice on your system works like a different program and has quirks that cause it to adhere more or less to published standards. 

Microsoft Eva was never released for general use like the other Microsoft voices and may not support the same feature set as more fully released voices. Given that Microsoft Eva isn't a voice that Microsoft supports outside of their Cortana app, we determined that we couldn't provide the same level of support for it that we could for other Microsoft voices.

I'll give this some thought but no promises.

**Comment by Ekristoffe** [comment]
Indeed, I am shocked haven't found this one...
Yes the solution is simple as adding a "," (haven't tested with a ".") at the end of the pause line.
I think this could be easily added in the future for maybe voice to come.
Anyway I have found and corrected for myself I just wanted to share the community my findings and maybe improve EDDI for non microsoft voices (in case of 3rd party voices) or yet unreleased voices.
Personally I have found EVA more human like talking and this is the reason I use a heavily modified script with her.
Anyway thanks.
(Closing it since I don't think it is needed anymore unless we can add a FAQ point for this.)
